client/src/common/components/GridList.jsx

```diff
@@ -91,6 +91,7 @@ class GridList extends PureComponent {
                   color={color}
                   description={item.description}
                   isFavourite={item.isFavourite}
+                  isPrivate={item.isPrivate}
```

Comment:
Stop using `is` prefix on props names.

Reply:
![Zrzut ekranu 2019-04-01 o 16 41 48](https://user-images.githubusercontent.com/27632432/55336355-13c83f80-549d-11e9-89f6-eaf758bbec13.png)
I can't use only `private` word cause eslint says it is `private is a reserved word in strict mode`
How I can name it?